packages/block-editor/src/components/colors-gradients/style.scss

```diff
@@ -38,4 +38,29 @@
 	.block-editor-block-inspector & .components-base-control {
 		margin-bottom: inherit;
 	}
+
+	.components-dropdown {
+		display: block;
+	}
+
+	// Allow horizontal overflow so the size-increasing color indicators don't cause a scrollbar.
+	.components-navigator-screen {
+		overflow-x: visible;
```

Comment:
As @jasmussen mentioned [here](https://github.com/WordPress/gutenberg/pull/37053#discussion_r761188224), we should be able to remove this rule safely